src/systems/filecoin_mining/storage_mining/storage_miner_actor.go

```diff
@@ -72,71 +97,76 @@ func (sm *StorageMinerActor_I) submitFaultReport(
 		NewTerminatedFaults_: newTerminatedFaults,
 	}
 
-	sm.SectorTable().Impl().TerminationFaultCount_ = util.UVarint(0)
-
 	panic(faultReport)
```

Comment:
- i think `panic` makes the go compiler ignore some errors below the `panic(.)` statement
- maybe this should be just a `// TODO` ?

Reply:
Modifying these to use `rt.Fatal(.)` instead of `panic(.)`.

Reply:
ok sounds good -- looks like rt.Fatal is now there, but the panic is still there. should remove the panics, as they are hiding compiler errors in this function